src/main/resources/mustache-templates/sdh.mustache

```diff
@@ -2,19 +2,21 @@
 
 <div class="container vertical-space">
 	{{#patient}}
+		<div class="float-right"><a href="#" class="btn btn-primary"><i class="fas fa-envelope"></i> Send Message</a></div>		
 		<h3>Patient Info</h3> 
-		Name: {{#name}}{{.}}{{/name}}<br>
-		Date of Birth: {{#dateOfBirth}}{{.}}{{/dateOfBirth}}<br>		
+		Name: <a href="{{req.contextPath}}/patient/{{#id}}{{.}}{{/id}}">{{#name}}{{.}}{{/name}}</a><br>
+		Date of Birth: {{#dateOfBirth}}{{.}}{{/dateOfBirth}}<br>
 	{{/patient}}
 	<br>
-
+	
 	<h3>Social Determinants of Health</h3>
 	
 	<div id="contents">
 		<b-table show-empty head-variant="dark" :items="emphasized" :fields="fields">
 		  <template slot="top-row" slot-scope="{ fields }">
 		    <td v-for="field in fields" :key="field.key">
-		      <input v-model="filters[field.key]" :placeholder="field.label">
+		      <span v-if="field.key === 'show_details'" >Filter By:</span>
+		      <input v-if="field.key !== 'show_details'" v-model="filters[field.key]" :placeholder="field.label">
```

Comment:
This seems like a case where you would want to use [`v-else`](https://vuejs.org/v2/guide/conditional.html#v-else). Any particular reason you're not? I also like to keep JavaScript logic out of Vue templates, so I would probably use a computed property here, even though it's just a simple comparison.

Reply:
Actually, now that I look at this harder, a CP isn't appropriate due to the iteration. You could eliminate the logic by calling a method, though.

Reply:
I couldn't get the method call working properly, so maybe we can look at that together. I addressed all other comments. Thanks!

Reply:
I ended up adding some logic using a method. I was originally trying to have the method pass back the html to render, and that was not working. Instead I pass back a boolean used to compute which component to show.